### PR TITLE
Integrate EEGReader with CMLReader

### PR DESCRIPTION
**Pull request by mivade**
- [x] Fix issues with defining time coordinates
- [x] Make `EEGReader` work with `CMLReader`
- [x] Allow reading epochs via events
- [ ] Store events in resulting `TimeSeries`

### CODE REVIEW

**Comment by mivade** [comment]
When specifying events for reading EEG data, the current behavior is to default relative stop times to be 100 ms after event onset. Thoughts on if we should use a default value here or should we make this a required keyword argument?

**Comment by zduey** [comment]
My inclination is to make it required since I don’t know of a reasonable default since most use cases will be very analysis specific. One of the grad students or post docs could probably confirm.

File: cmlreaders/base_reader.py [hunk 1]
```diff
@@ -54,15 +54,21 @@ def __init__(self, data_type: str, subject: Optional[str] = None,
 
         self._file_path = file_path
 
-        # When no file path is given, look it up using PathFinder
-        if file_path is None:
+        # When no file path is given, look it up using PathFinder unless we're
+        # loading EEG data
```
Comment (zduey) [inline cmlreaders/base_reader.py]: might be worth giving a brief explanation as to why EEG is a special snowflake

File: cmlreaders/cmlreader.py [hunk 2]
```diff
@@ -101,12 +101,12 @@ def load_eeg(self, events: Optional[pd.DataFrame] = None,
...
             Events to load EEG epochs from. Incompatible with passing
             ``epochs``.
-        pre
-            Time in ms to include prior to passed events (default: 0). This
-            parameter has no effect if events are not specified.
-        post
-            Time in ms to include after passed events (default: 0). This
-            parameter has no effect if events are not specified.
+        rel_start
+            Start time in ms relative to passed event onsets (default: 0). This
```
Comment (zduey) [inline cmlreaders/cmlreader.py]: looks like default is no longer 0

File: cmlreaders/cmlreader.py [hunk 3]
```diff
@@ -101,12 +101,12 @@ def load_eeg(self, events: Optional[pd.DataFrame] = None,
...
+            parameter has no effect if epochs are specified instead of events.
+        rel_stop
+            Stop time in ms relative to passed event onsets (default: 100). This
```
Comment (zduey) [inline cmlreaders/cmlreader.py]: wrong default as well

File: cmlreaders/cmlreader.py [hunk 4]
```diff
@@ -140,10 +140,14 @@ def load_eeg(self, events: Optional[pd.DataFrame] = None,
         }
 
         if events is not None:
+            if rel_start is None or rel_stop is None:
+                raise ValueError("rel_start and rel_stop are required keyword"
```
Comment (zduey) [inline cmlreaders/cmlreader.py]: May be worth raising IncompatibleParametersError as defined in cmlreaders/exc.py instead

**Comment by mivade** [comment]
I'm going to ignore the remaining TODO item about events because I don't care about events being in the `TimeSeries` that is returned right now. Others are free to implement this and submit a new PR.

**Pull request merged**